app/models/institution.ts

```diff
@@ -85,6 +85,10 @@ export default class InstitutionModel extends OsfModel {
     }
 }
 
+export interface PreprintInstitutionModel extends InstitutionModel {
+    isSelected: boolean;
+}
+
```

Comment:
Minor nit, I don't feel like this belongs in the institution model itself, but more appropriate to be in the institution-manager component